src/gateway/get-started/proxy-caching.md

```diff
@@ -1,73 +1,190 @@
+   |Bypass| The request could not be satisfied from cache based on plugin configuration.|
+
+### Service level proxy caching
+
+The Proxy Cache plugin can be enabled for specific services. The request is the same as above, but posted to the service URL:
+
+```sh
+curl -X POST http://localhost:8001/services/example_service/plugins \
+   --data "name=proxy-cache" \
+   --data "config.request_method=GET" \
+   --data "config.response_code=200" \
+   --data "config.content_type=application/json; charset=utf-8" \
+   --data "config.cache_ttl=30" \
+   --data "config.strategy=memory"
 ```
 
-Depending on your configuration, the response header will be composed of many different fields. Notice the integer values in the following fields:
+### Route level proxy caching
 
-* `X-Kong-Proxy-Latency`
+The Rate Limiting plugin can be enabled for specific routes. The request is the same as above, but posted to the route URL:
 
-* `X-Kong-Upstream-Latency`
+```sh
+$ curl -X POST http://localhost:8001/routes/mock/plugins \
+   --data "name=proxy-cache" \
+   --data "config.request_method=GET" \
+   --data "config.response_code=200" \
+   --data "config.content_type=application/json; charset=utf-8" \
+   --data "config.cache_ttl=30" \
+   --data "config.strategy=memory"
+```
 
-If you were to send two requests in succession, the values would be lower in the second request. That demonstrates that the content is cached, and {{site.base_gateway}} is not returning the information directly from the upstream service that your route is pointing to.
+### Consumer level proxy caching
 
-* `X-Cache-Status`
+In {{site.base_gateway}}, [consumers](/gateway/latest/admin-api/#consumer-object) are an abstraction that defines a user of a service. 
+Consumer-level proxy caching can be used to cache responses per consumer.
 
-This will display `hit` expressing that proxy caching worked correctly, but this header can also return the following states:
+1. **Create a consumer** 
 
-|State| Description|
-|---|---|
-|Miss| The request could be satisfied in cache, but an entry for the resource was not found in cache, and the request was proxied upstream.|
-|Hit| The request could be satisfied in cache, but an entry for the resource was not found in cache, and the request was proxied upstream.|
-|Refresh| The resource was found in cache, but could not satisfy the request, due to Cache-Control behaviors or reaching its hard-coded `cache_ttl` threshold.|
-|Bypass| The request could not be satisfied from cache based on plugin configuration.|
+Consumers are created using the consumer object in the Admin API.
 
-In the initial request the value for `config.content_type` was set to "application/json". The proxy cache plugin will only cache the specific data type that was set in the initial configuration request. 
+```sh
+curl -X POST http://localhost:8001/consumers/ \
+  --data username=dfossey
```

Comment:
```suggestion
  --data username=example_consumer
```

Reply:
I specifically decided to use unique username in this page. That way the user is not exposed to potential conflicts as they progress through the entire tutorial.  I choose a famous primatologist here which also adds some interest to the document, a break from the mundane

Reply:
We have been working on removing variables like this and replacing them with generic names and values. 

The style guide doesn't specify this exact case, but you can see our intention to move in this direction.
https://release-gateway-3-0--kongdocs.netlify.app/contributing/style-guide/#placeholder-and-example-values

Reply:
see revised username